src/Elmish.WPF/Elmish.WPF.fsproj

```diff
@@ -52,11 +52,11 @@
 
   <ItemGroup>
     <PackageReference Include="Elmish" Version="[3.1.0, 3.99]" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[3.1.6, 99]" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[6.0.1, 99]" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="[4.6.2, 99]" />
+    <PackageReference Update="FSharp.Core" Version="[6.0.5, 99]" />
```

Comment:
Did we have to make this change?  It is causing a problem described in #509.

Reply:
We were using some of the newer features in some of the code, but I'm sure there are workarounds. I don't remember specifics though.